tester/paddle_to_torch/mapping.json

```
@@ -6114,10 +6123,35 @@
         "min_input_args": 1
     },
     "paddle.sort": {
-        "Rule": "SortRule"
+        "Rule": "SortRule",
+        "torch_api": "torch.sort",
+        "set_defaults": {
+            "axis": "-1",
+            "stable": "False",
+            "descending": "False",
+            "name": "None"
```

"False" 也可直接写作 false
```suggestion
            "descending": "False"
```